### DISCUSSION TITLE
Is there any Debug UI project?

### DISCUSSION THREAD

**Discussion by 1oglop1** [renovatebot/renovate#25749] [Request Help]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

selfhosted - gitlab

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate produces `.ndjson` logs which are usually very long. Reading these for debugging purpose is quite difficult with just plain IDE/text editor.

It would be great to have some UI that can display logs nicely structured.

I wonder if anyone tried to write a project like this already?

### What you tried so far.

* Search text in ndjson
* Take one line into a new file
* examine that line

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Renovate defaults to pretty formatted logs in stdout

**Reply by 1oglop1** [reply]
I know it does, but I often need detailed information to read `depnames` package names etc.
the INFO log level is not enough, I do wonder if DEBUG level would clutter the CI output too much.
```
"deps": [
          {
            "currentValue": "18.18.1",
            "currentVersion": "18.18.1",
            "datasource": "node-version",
            "depName": "node",
            "fixedVersion": "18.18.1",
            "homepage": "https://nodejs.org",
            "isSingleVersion": true,
            "packageName": "node",
            "registryUrl": "https://nodejs.org/dist",
            "sourceUrl": "https://github.com/nodejs/node",
            "versioning": "node",
            "warnings": [],
            "updates":
 ```

**Reply by rarkins** [reply]
We always use debug
